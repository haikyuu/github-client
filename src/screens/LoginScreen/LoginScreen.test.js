import React from 'react'
import renderer from 'react-test-renderer'
import { LoginScreen } from '@screens'

it('renders the LoginScreen from snapshot', () => {
  expect(renderer.create(
    <LoginScreen />
  )).toMatchSnapshot()
})

it('renders the message given', () => {

	const component = renderer.create(
    	<LoginScreen />
	)
	const response = 'token'
	component.getInstance().displayResponse(response)
	expect(component).toMatchSnapshot()
})
