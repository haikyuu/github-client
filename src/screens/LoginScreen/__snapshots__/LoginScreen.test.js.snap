// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the LoginScreen from snapshot 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Please Login to your Github account.
  </Text>
  <View
    style={
      Object {
        "padding": 10,
      }
    }
  >
    <TextInput
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Username"
      style={
        Object {
          "borderColor": "gray",
          "borderWidth": 1,
          "height": 40,
        }
      }
      value=""
    />
    <TextInput
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Password"
      secureTextEntry={true}
      style={
        Object {
          "borderColor": "gray",
          "borderWidth": 1,
          "height": 40,
          "marginTop": 5,
        }
      }
      value=""
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#841584",
              },
            ]
          }
        >
          Login
        </Text>
      </View>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    />
  </View>
</View>
`;

exports[`renders the message given 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Please Login to your Github account.
  </Text>
  <View
    style={
      Object {
        "padding": 10,
      }
    }
  >
    <TextInput
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Username"
      style={
        Object {
          "borderColor": "gray",
          "borderWidth": 1,
          "height": 40,
        }
      }
      value=""
    />
    <TextInput
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Password"
      secureTextEntry={true}
      style={
        Object {
          "borderColor": "gray",
          "borderWidth": 1,
          "height": 40,
          "marginTop": 5,
        }
      }
      value=""
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#841584",
              },
            ]
          }
        >
          Login
        </Text>
      </View>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      token
    </Text>
  </View>
</View>
`;
